docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '2'
```

Comment:
while working on this fix I received local warnings about a deprecated version parameter

Reply:
The "version" is used to declare file format it's only used for Compose v1, Compose v2 will display a warning and this setting be ignored. I know the intention is good, but his is out of scope and it may be used in some cases, lets remove this change.